.goreleaser.yaml

```diff
@@ -0,0 +1,201 @@
+project_name: sbom-convert
+before:
+  hooks:
+    - go mod tidy
+    - ./scripts/completions.sh {{ .ProjectName }}
+    - ./scripts/manpages.sh {{ .ProjectName }}
+    - go generate ./...
+
+builds:
+  - env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
+      - darwin
+    goarch:
+      - "386"
+      - amd64
+      - arm
+      - arm64
+    goarm:
+      - "7"
+    ignore:
+      - goos: windows
+        goarch: arm
+    flags:
+      - -trimpath
+    ldflags:
+      - -s -w -X {{ .ModulePath }}/cmd/cli.version={{.Version}}
+    main: .
+checksum:
+  name_template: 'checksums.txt'
+snapshot:
+  name_template: "{{ incpatch .Version }}-next"
+
+# TODO: improve section: ref: https://github.com/goreleaser/goreleaser/blob/main/.goreleaser.yaml
+changelog:
+  sort: asc
+  filters:
+    exclude:
+      - '^docs:'
+      - '^test:'
+      - "merge conflict"
+      - Merge pull request
+      - Merge remote-tracking branch
+      - Merge branch
+      - go mod tidy
+
+release:
+ disable: false
+ name_template: 'v{{ .Version }}'
```

Comment:
if i understand correctly you tag the repo with x.y.z and it creates a version with vX.Y..Z?

Reply:
version is without "v"  (if u do --version) but publicly, tags and releases have it.

Reply:
Can you elaborate a bit about the standard or spec behind this?

If i understand your saying i will tag `git tag v0.0.1` and the images,git tags and releases artifacts will have v0.0.0 right so its basically only the version command that will have the unprefixed version, if this is the case i have no issue here , resolve the comment.

I just say my two sense on this.
Personally i prefer consistency, looking forward some CDX & SPDX formats allow you to include the tool identification creating them (its probably not required) .
It may be valuable to have consistency with the public artifact versing scheme.
Now after all this blaber i guess we will simply add the `v` if need be..

Reply:
We can streamline that no problem, I'm fine with going for v0.0.0 everywhere. I'll just add `v` in front of the version :D

Reply:
![image](https://github.com/bom-squad/go-cli/assets/116580393/f344ba8d-4221-4488-adf5-3d8cb8365bf5)